troi/patches/daily_jams.py

```diff
@@ -19,20 +19,21 @@ class DailyJamsElement(Element):
         Split weekly recommended recordings into 7 sets, one for each day of the week.
```

Comment:
what do we think about putting Elements in the `patches` module?  If they're only used in this patch, I guess this isn't too much of an issue. Can we foresee this element being used elsewhere in the future? In that case would we move it?

Reply:
My plan was to leave them in the patches until it was clear it would be useful in another context and then move it.

Reply:
fine by me